README.md

```diff
@@ -1,9 +1,15 @@
-# rentreceipt
+# Rent Receipt Generator
 Rent Receipt for Employees in India
 
 To create your Rent Receipt pdf file:
 
-1. Install LaTeX.
-2. Modify the rentreceipt.tex and execute:
+### Prerequisites
+1. Install texlive-relsize.noarch package
+2. Install LaTeX.
```

Comment:
It would be better if you could specify it in the following order. 
```
1. The command you executed for the basic latex installation.
2. command for `tfrupee.sty` installation.
3. Command for `texlive-relsize` installation.
```
You can even combine these to a single command. (I cannot do this, as latex is already installed in all my machines for other intentions).